src/modules/paymentProcessor/PP_Streaming_v1.sol

```diff
@@ -33,6 +33,16 @@ import {SafeERC20} from "@oz/token/ERC20/utils/SafeERC20.sol";
  *          multiple clients and recipients, integrated with error handling for
  *          payments and managing active streaming schedules and their cancellations.
  *
+ *          DISCLAIMER: Known Limitations
```

Comment:
Maybe explain in the note that this happens for paymentClients where users can trigger payment streams by themselves (like the stakingManager) and specifically warn of using this processor in that type of setup?